conf/application.conf

```diff
@@ -154,4 +154,6 @@ nunjucks {
 
 features {
   send-email = false
-}
\ No newline at end of file
+}
+
+contactDetailsToggle = true
```

Comment:
either set this to false here or false in app-config-base. This isn't a fully designed end to end journey yet is it?